### ISSUE TITLE
BART: Denoising Sequence-to-Sequence Pre-training for Natural Language Generation, Translation, and Comprehension

### ISSUE THREAD

**Issue by jinmang2** [jiphyeonjeon/season3#7]
## 집현전 중급반 스터디
- 2022년 4월 03일 일요일 9시
- 안형준님 박설현님 노은지님 발표
- 논문 링크: https://arxiv.org/abs/1910.13461

> ### Abstract
> We present BART, a denoising autoencoder for pretraining sequence-to-sequence models. BART is trained by (1) corrupting text with an arbitrary noising function, and (2) learning a model to reconstruct the original text. It uses a standard Tranformer-based neural machine translation architecture which, despite its simplicity, can be seen as generalizing BERT (due to the bidirectional encoder), GPT (with the left-to-right decoder), and many other more recent pretraining schemes. We evaluate a number of noising approaches, finding the best performance by both randomly shuffling the order of the original sentences and using a novel in-filling scheme, where spans of text are replaced with a single mask token. BART is particularly effective when fine tuned for text generation but also works well for comprehension tasks. It matches the performance of RoBERTa with comparable training resources on GLUE and SQuAD, achieves new state-of-the-art results on a range of abstractive dialogue, question answering, and summarization tasks, with gains of up to 6 ROUGE. BART also provides a 1.1 BLEU increase over a back-translation system for machine translation, with only target language pretraining. We also report ablation experiments that replicate other pretraining schemes within the BART framework, to better measure which factors most influence end-task performance.

**Comment by jinmang2** [comment]
- 발표 링크: https://www.youtube.com/watch?v=OmLzxmkfzH4&list=PL2tRglRS_GqjCcBVM26gYNdoOnX12lLIh&index=3&t=11s
